### PR TITLE
add docs of get_image_backend, set_image_backend, image_load

### PR DESCRIPTION
**Pull request by LielinJiang**
add docs of get_image_backend, set_image_backend, image_load

### CODE REVIEW

File: doc/paddle/api/paddle/vision/image/get_image_backend_cn.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.. _cn_api_vision_image_get_image_backend:
+
+get_image_backend
+-------------------------------
+
+.. py:function:: paddle.vision.image.get_image_backend()
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/vision/image/get_image_backend_cn.rst:6]: paddle.vision.image.get_image_backend -> paddle.vision.get_image_backend
在 [alias_api_mapping](https://github.com/PaddlePaddle/FluidDoc/blob/develop/doc/paddle/api/alias_api_mapping)中定义alias

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst [hunk 2]
```diff
@@ -0,0 +1,48 @@
+.. _cn_api_vision_image_image_load:
+
+image_load
+-------------------------------
+
+.. py:function:: paddle.vision.image.image_load(path, backend=None)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/vision/image/image_load_cn.rst:6]: 同上

File: doc/paddle/api/paddle/vision/image/set_image_backend_cn.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.. _cn_api_vision_image_set_image_backend:
+
+set_image_backend
+-------------------------------
+
+.. py:function:: paddle.vision.image.set_image_backend(backend)
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/vision/image/set_image_backend_cn.rst:6]: 同上

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+读取一个图像。
+
+参数
+:::::::::
+
+    - path (PIL.Image|np.array) - 图像路径。
+    - backend (str): 加载图像的后端。 参数必须是 ``cv2``， ``pil``， ``None`` 之一。如果后端为 ``None`` ，则使用全局的 ``_imread_backend`` 参数，默认值为 ``pil`` 。 这个参数可以使用``paddle.vision.set_image_backend''指定。 默认值：None 。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/vision/image/image_load_cn.rst]: backend (str) -> backend (str，可选)

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+-------------------------------
+
+.. py:function:: paddle.vision.image.image_load(path, backend=None)
+
+读取一个图像。
+
+参数
+:::::::::
+
+    - path (PIL.Image|np.array) - 图像路径。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/vision/image/image_load_cn.rst]: path (PIL.Image|np.array) -> path (str)

File: doc/paddle/api/paddle/vision/image/get_image_backend_cn.rst (hunk 1 above)
Comment (LielinJiang) [reply doc/paddle/api/paddle/vision/image/get_image_backend_cn.rst:6]: done, thanks

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst (hunk 2 above)
Comment (LielinJiang) [reply doc/paddle/api/paddle/vision/image/image_load_cn.rst:6]: done, thanks

File: doc/paddle/api/paddle/vision/image/set_image_backend_cn.rst (hunk 3 above)
Comment (LielinJiang) [reply doc/paddle/api/paddle/vision/image/set_image_backend_cn.rst:6]: done, thanks

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst (hunk 4 above)
Comment (LielinJiang) [reply doc/paddle/api/paddle/vision/image/image_load_cn.rst]: done, thanks

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst (hunk 5 above)
Comment (LielinJiang) [reply doc/paddle/api/paddle/vision/image/image_load_cn.rst]: done, thanks

File: doc/paddle/api/paddle/vision/image/image_load_cn.rst [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+.. py:function:: paddle.vision.image.image_load(path, backend=None)
+
+读取一个图像。
+
+参数
+:::::::::
+
+    - path (str) - 图像路径。
+    - backend (str，可选): 加载图像的后端。 参数必须是 ``cv2``， ``pil``， ``None`` 之一。如果后端为 ``None`` ，则使用全局的 ``_imread_backend`` 参数，默认值为 ``pil`` 。 这个参数可以使用``paddle.vision.set_image_backend''指定。 默认值：None 。
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/vision/image/image_load_cn.rst]: 这个参数可以使用【``paddle.vision.set_image_backend''】格式有问题

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**